config/locales/en.yml

```diff
@@ -827,6 +827,12 @@ en:
     feedback_email: "FE-Levellingup.PremiumPayments@education.gov.uk"
     support_email_address: "FE-Levellingup.PremiumPayments@education.gov.uk"
     claim_subject: "Further education payment"
+    policy_acronym: FE
+    policy_short_name: Further education payments
```

Comment:
In figma I think this is now "Targeted Retention Incentive Payment For FE Teachers"

Reply:
it's called `policy_short_name` 😂